xbmc/cores/VideoPlayer/DVDSubtitles/webvtt/WebVTTHandler.cpp

```diff
@@ -1145,6 +1145,6 @@ double CWebVTTHandler::GetTimeFromRegexTS(CRegExp& regex, int indexStart /* = 1
   if (!regex.GetMatch(indexStart).empty())
     sHours = std::stoi(regex.GetMatch(indexStart).c_str());
   int sMinutes = std::stoi(regex.GetMatch(indexStart + 1).c_str());
-  int sSeconds = std::atof(regex.GetMatch(indexStart + 2).c_str());
+  double sSeconds = std::atof(regex.GetMatch(indexStart + 2).c_str());
```

Comment:
maybe `std::stof` to avoid the conversion to c string?

Reply:
oh yes yourright